src/clients/c++/perf_client/perf_client.cc

```diff
@@ -243,6 +243,8 @@ Usage(char** argv, const std::string& msg = std::string())
   std::cerr << "II. INPUT DATA OPTIONS: " << std::endl;
   std::cerr << "\t-b <batch size>" << std::endl;
   std::cerr << "\t--input-data <\"zero\"|\"random\"|<path>>" << std::endl;
+  std::cerr << "\t--shared-memory <\"system\"|\"cuda\"|\"none\">" << std::endl;
+  std::cerr << "\t--output-shm-size <size in bytes>" << std::endl;
```

Comment:
We've avoided using shm in external "controls". How about --output-shared-memory-size.